index.js

```diff
@@ -21,14 +21,17 @@ var toArray = Array.from || (function (slice) {
 
 // ===================================================================
 
-function eventToPromise (emitter, event, opts) {
-  var ignoreErrors = opts && opts.ignoreErrors
+function eventToPromise (emitter, event, _opts) {
+  var opts = _opts || {}
+  var ignoreErrors = opts.ignoreErrors
+  var errorEvent = opts.error || 'error'
```

Comment:
Could do, if you don't mind a breaking change :-) I personally prefer an explicit option, though, even if it is slightly "longer", since it means I don't have to add a comment explaining that `error: null` means "ignore errors" :-)